### DISCUSSION TITLE
Can we customize bootstrap color? how?

### DISCUSSION THREAD

**Discussion by shvm-99** [twbs/bootstrap#39218] [General] [answered]
I have tried but some errors occured?please let me know?

**Comment by SHVM-09** [comment, marked as answer]
Here is docs link for bootstrap follow it
https://getbootstrap.com/docs/5.0/customize/color/
Plus make sure you keep everything in correct order and pass on custom color to color mixin and you will be able t do it!

**Comment by shvm-99** [comment]
ty
